nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/JobCleanUpService.java

```diff
@@ -100,21 +100,20 @@
     public void postConstruct() {
         logger.log(Level.FINE,KernelLoggerInfo.initializingJobCleanup);
 
-
+        if (Version.getFullVersion().contains("Micro")) {
```

Comment:
The scheduler is also *not* used in Payara Micro so no need to create it.

Reply:
Shouldn't the log output also be affected by the guard?

Reply:
I agree with Jan here, the logger does not log "postConstruct" entry here, but on Micro lies that it is initializing the cleanup service, but it does not do that.

Reply:
Btw on line 151 cleanupFuture variable resulting from scheduling can be removed.

Reply:
@dmatej Correct regarding the cleanupFuture variable

But there are a lot more code 'issues'. (12 in total) Unused injection point, unused class variables, unused method parameter, unnecessary return types, unnecessary switch  etc ...

But cleanup of the class should not be the goal of this PR. (as it will hide later on what actually was changed)

Reply:
I agree, that was only first thing I noticed. Other things may have side effects, so should be postponed.